README.md

```diff
@@ -1,24 +1,17 @@
-# README
+# rails-async-websocket-pubsub-example
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Development
 
-Things you may want to cover:
+```
+$ bundle ex foreman start
+$ websocat ws://localhost:3001
+$ websocat ws://localhost:3002
+```
 
-* Ruby version
+![](demo.gif)
 
-* System dependencies
+## Production
 
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+```
+$ bundle exec falcon host
+```
```

Comment:
In a production environment, is it sufficient to simply start with bundle exec falcon host regardless of the server's CPU core count? Even if a large number of Fibers are created, will there still only be a single Ruby process?

Reply:
By default `bundle exec falcon host` will start one server process per processor core, and one event loop per server process, which can handle multiple inbound requests.

If you only want a single process, you will need to configure `count` in your `falcon.rb` file <https://socketry.github.io/falcon/source/Falcon/Environment/Server/index.html#Falcon::Environment::Server#count>.